routers/api/v1/repo/pull_review.go

```diff
@@ -0,0 +1,520 @@
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/PullReviewCommentList"
+	//   "404":
+	//     "$ref": "#/responses/notFound"
+
+	review, _, statusSet := prepareSingleReview(ctx)
+	if statusSet {
+		return
+	}
+
+	apiComments, err := convert.ToPullReviewCommentList(review, ctx.User)
+	if err != nil {
+		ctx.Error(http.StatusInternalServerError, "convertToPullReviewCommentList", err)
+		return
+	}
+
+	ctx.JSON(http.StatusOK, apiComments)
+}
+
+// DeletePullReview delete a specific review from a pull request
+func DeletePullReview(ctx *context.APIContext) {
+	// swagger:operation DELETE /repos/{owner}/{repo}/pulls/{index}/reviews/{id} repository repoDeletePullReview
+	// ---
+	// summary: Delete a specific review from a pull request
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: owner
+	//   in: path
+	//   description: owner of the repo
+	//   type: string
+	//   required: true
+	// - name: repo
+	//   in: path
+	//   description: name of the repo
+	//   type: string
+	//   required: true
+	// - name: index
+	//   in: path
+	//   description: index of the pull request
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// - name: id
+	//   in: path
+	//   description: id of the review
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// responses:
+	//   "204":
+	//     "$ref": "#/responses/empty"
+	//   "403":
+	//     "$ref": "#/responses/forbidden"
+	//   "404":
+	//     "$ref": "#/responses/notFound"
+
+	review, _, statusSet := prepareSingleReview(ctx)
+	if statusSet {
+		return
+	}
+
+	if ctx.User == nil {
+		ctx.NotFound()
+		return
+	}
+	if !ctx.User.IsAdmin && ctx.User.ID != review.ReviewerID {
+		ctx.Error(http.StatusForbidden, "only admin and user itself can delete a review", nil)
+		return
+	}
+
+	if err := models.DeleteReview(review); err != nil {
+		ctx.Error(http.StatusInternalServerError, "DeleteReview", fmt.Errorf("can not delete ReviewID: %d", review.ID))
+		return
+	}
+
+	ctx.Status(http.StatusNoContent)
+}
+
+// CreatePullReview create a review to an pull request
+func CreatePullReview(ctx *context.APIContext, opts api.CreatePullReviewOptions) {
+	// swagger:operation POST /repos/{owner}/{repo}/pulls/{index}/reviews repository repoCreatePullReview
+	// ---
+	// summary: Create a review to an pull request
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: owner
+	//   in: path
+	//   description: owner of the repo
+	//   type: string
+	//   required: true
+	// - name: repo
+	//   in: path
+	//   description: name of the repo
+	//   type: string
+	//   required: true
+	// - name: index
+	//   in: path
+	//   description: index of the pull request
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// - name: body
+	//   in: body
+	//   required: true
+	//   schema:
+	//     "$ref": "#/definitions/CreatePullReviewOptions"
+	// responses:
+	//   "200":
```

Comment:
201 for created?

Reply:
I used 200 only to follow github's documentation - but I'm up to change it again ... just need some consencus

Reply:
Ok, I guess following GH is a valid point.

Reply:
yes ... I personaly go with 201 too  but I wrote down the guidelines for the API so I should follow them somehow ;)